miniwallet/test/Mini1155.ts

```diff
@@ -0,0 +1,489 @@
+import { expect } from 'chai'
```

Comment:
Is there any way we can continue the review of these files in the other PR where they were originally reviewed? Several hours were spent reviewing the files, so it would be nice to continue from existing reviewing checkpoint, rather than starting from scratch

Reply:
I went through the code again this time. Let's keep reviewed PRs open in the future